README.md

```diff
@@ -250,4 +250,57 @@ POST
     - if course is added successfully
         - `{"success": true, "message": "New Course successfully saved."}`
     - if course could not be added
-        - `{"success": false, "msg": "Unexpected error."}`
\ No newline at end of file
+        - `{"success": false, "msg": "Unexpected error."}`
+
+**/updateTeams**  
+
+POST
+- When the user is logged in and selects 'save teams' invoke gets data from new applicant form and passes into insertNewApplicantToDb function for inserting into database.
+- Returns success true / false
+  - If successful it returns 'success true' with message 'Team successfully saved' 
+  - If trainer name is missing it returns success => false with message 'Missing trainer names'
+
+**/getCourses**  
+
+GET
+- Calls a method to get all the courses scheduled and send JSON back with the info
```

Comment:
Again, look at the existing documentation: most descriptions are along the lines of: "Get all the scheduled courses from the courses table in the database", which is short and clear.